v2/support/instances_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<body>
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    var client = ZAFClient.init();
+
+    function commentTemplate(currentUser, requester, lineBreak) {
+      return 'Hi ' + currentUser + ',' + lineBreak + lineBreak + 'Kind regards,' + lineBreak + requester;
+    }
+
+    // instance.created fires whenever an instance of your app is created
+    client.on('instance.created', function(context) {
+      // handle all ticket_sidebar instances
+      if (context.location === 'ticket_sidebar') {
+        // create a new ZAFClient for the ticket sidebar instance
+        var ticketInstance = client.instance(context.instanceGuid);
+
+        ticketInstance.get([
+          'currentUser.name',
+          'ticket.requester.name',
+          'comment'
+        ]).then(function(data) {
+          var currentUser = data['currentUser.name'],
+              requester = data['ticket.requester.name'],
+              comment = data['comment'],
+              lineBreak = comment.useRichText ? '<br />' : '\n';
+
+          if (comment.text === '' || comment.text === '<p></p>') {
```

Comment:
yep, more naive empty checks have broken in the past because of a stray whitspace or <br> included in the rich text version though, haven't they? This code will serve as an example of this functionality too.
That said, I'm not convinced its actually worth it.